openesdh-repo/src/main/java/dk/openesdh/repo/webscripts/files/OeFilesWebScript.java

```diff
@@ -120,12 +120,12 @@ public void move(
 
     @Uri(value = "/api/openesdh/case/{caseId}/addFile", method = HttpMethod.PUT, defaultFormat = JSON)
     public void addFileToCase(
-            @UriVariable("caseId") final String caseId,
-            @RequestParam("nodeRef") final NodeRef file,
-            @RequestParam("title") final String title,
-            @RequestParam("doc_type") final NodeRef docType,
-            @RequestParam("doc_category") final NodeRef docCategory,
-            @RequestParam(value = "description", required = false) final String description) {
+            @UriVariable(WebScriptUtils.CASE_ID) final String caseId,
```

Comment:
those are webscript specific casesensitive parameter names, i'm not sure if it is ok to extract to WebScriptUtils